### PR TITLE
Add ci example

### PR DESCRIPTION
**Pull request by mkam039**

### CODE REVIEW

File: docs/ci_example.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+- `SERVICE_ACCOUNT_KEY`: contents of a `keyfile.json`. For more information see [here](https://docs.github.com/en/actions/security-guides/encrypted-secrets)
+- `GCLOUD_PROJECT`: your Google Cloud project ID.
+- `BQ_DATASET`: name of the BigQuery dataset
+
+If your fal scripts require environment variables, these should also be provided as repository secrets.
+
+## Setup dbt project
+
+### `profiles.yml`
```
Comment (chamini2) [inline docs/ci_example.md:15]: Is this file to be commited in the repo?

File: docs/ci_example.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  with:
+    python-version: '3.8.x'
+
+- name: Install dependencies
+  run: |
+    pip install --upgrade --upgrade-strategy eager -r requirements.txt
+```
+
+### Setup dbt profile
+Next, we setup `profiles.yml`:
```
Comment (chamini2) [inline docs/ci_example.md:66]: You could use `dbt --profile-dir` flag instead

File: docs/ci_example.md (hunk 1 above)
Comment (mkam039) [reply docs/ci_example.md:15]: yes

File: docs/ci_example.md (hunk 2 above)
Comment (mkam039) [reply docs/ci_example.md:66]: profile-dir is still going to be relative, so why not just put it in ~/.dbt

Comment (mkam039) [reply docs/ci_example.md:66]: Also, every dbt command that we would need to run in a workflow would require this flag. I think it's much easier to just write `dbt run` or `dbt seed`.

Comment (chamini2) [reply docs/ci_example.md:66]: I think it's just an opinion / preference thing. So either works!

**Pull request merged**